src/components/AttachmentViewer/styles.ts

```diff
@@ -129,7 +112,7 @@ export const NextButton = styled(StyledButton)`
 `
 
 export const StyledInteractiveImage = styled(InteractiveImage)`
-  margin: ${themeSpacing(23, 0, 5)};
-  max-height: calc(100vh - ${themeSpacing(28)});
+  margin: ${themeSpacing(6, 0, 0)};
+  max-height: calc(100vh - ${themeSpacing(75)});
```

Comment:
height in pixels

Reply:
This one I don't get. Everything is with themespacing?

Reply:
changed it.